lib/perl/Genome/Model/Tools/BioSamtools/AlignmentSummaryV2.pm

```diff
@@ -577,8 +577,13 @@ sub _prepare_split_sorted_bed_files {
         my $chrom = $1;
 
         unless ($open_handles{$chrom}) {
+            if(scalar keys %open_handles >= 100) {
+                $_->close foreach values %open_handles;
```

Comment:
A lot of our input files are not sorted at all and jump around from chromosome to chromosome.  I had also thought about sorting the file in advance of this loop, so we knew we only needed one filehandle at a time.